plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
```

Comment:
You need to add a GPL 3+ copyright header. (See other modules for the one-line version.)